index.js

```diff
@@ -1,13 +1,16 @@
 'use strict';
 const ansiEscapes = require('ansi-escapes');
 const cliCursor = require('cli-cursor');
+const termSize = require('term-size');
```

Comment:
You don't really need `term-size` here as we're only doing this interactively, so we can just use `process.stdout.columns`.